packages/angular_devkit/build_angular/src/tools/webpack/plugins/karma/karma-context.html

```diff
@@ -4,7 +4,7 @@
 Loaded within the iframe.
 Reloaded before every execution run.
 -->
-<html>
+<html lang="en">
```

Comment:
I do not think that this change is correct, as the language is not guaranteed to be `en`. As such in this case not having the `lang` attribute is better IMHO.

Reply:
Both of the karma files.